ui/tests/acceptance/clients/counts/overview-test.js

```diff
@@ -39,6 +39,12 @@ module('Acceptance | clients | overview', function (hooks) {
     timestamp.now.restore();
   });
 
+  test('it should render the correct tabs', async function (assert) {
+    assert.dom(SELECTORS.tab('overview')).exists();
+    assert.dom(SELECTORS.tab('token')).exists();
+    assert.dom(SELECTORS.tab('config')).exists();
```

Comment:
purposefully omitting making an assertion on the secrets sync tab for now. under this test setup it's hidden but the charts are still showing because the feature is considered active (since our test setup includes activity data but not DOES include the feature in the license). 

this doesn't seem like a realistic scenario; users (other than beta) won't have activity unless their license includes the feature. so we should circle back and make sure we're testing a realistic scenario.